src/mumble/MainWindow.cpp

```diff
@@ -1031,7 +1031,7 @@ void MainWindow::openUrl(const QUrl &url) {
 	QUrlQuery query(url);
 	QString version = query.queryItemValue(QLatin1String("version"));
 	if (version.size() > 0) {
-		if (!MumbleVersion::get(&major, &minor, &patch, version)) {
+		if (!Version::get(&major, &minor, &patch, version)) {
```

Comment:
Given that it is used for Mumble version representation, storage and conversion specifically I feel like the MumbleVersion name is more fitting than Version.

The logic also handles a int representation, which is probably specific to Mumbles version rather than a general version thing.

Reply:
This should be symbolized by using proper namespaces for the entire project in which case the complete namespace will be `Mumble::Version::get`. But I did not add that namespace yet as it is not used in other parts of Mumble yet.

Besides: The main reason for the rename is that there is a name clash for the name `MumbleVersion` as the version type used in the plugin API is named like that. It has to be prefixed with `Mumble` though as it can't be put inside a namespace as the API is C. And if it did not have the prefix clashes with other libraries are likely.

Reply:
If you have a better name than `Version` though, I'm all ears.

I was thinking that maybe `ProductVersion`, `ApplicationVersion` or `VersionUtils` ro something like this could be alternatives :thinking: